bitnami/supabase/templates/auth/default-configmap.yaml

```
@@ -9,8 +9,10 @@ kind: ConfigMap
 metadata:
   name: {{ printf "%s-default" (include "supabase.auth.fullname" .) | trunc 63 | trimSuffix "-" }}
   namespace: {{ include "common.names.namespace" . | quote }}
-  labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
-    app.kubernetes.io/part-of: supabase
+  {{- $versionLabel := dict "app.kubernetes.io/version" ( include "common.images.version" ( dict "imageRoot" .Values.auth.image "chart" .Chart ) ) }}
+  {{- $labels := include "common.tplvalues.merge" ( dict "values" ( list .Values.commonLabels $versionLabel ) "context" . ) }}
+  labels: {{- include "common.labels.standard" ( dict "customLabels" $labels "context" $ ) | nindent 4 }}
+      app.kubernetes.io/part-of: supabase
```

One indentation level less, right?
```suggestion
    app.kubernetes.io/part-of: supabase
```